crates/routes/src/feeds.rs

```diff
@@ -499,6 +499,7 @@ fn create_post_items(
     if let Some(url) = p.post.url {
       let link_html = format!("<br><a href=\"{url}\">{url}</a>");
       description.push_str(&link_html);
+      i.link(url.to_string());
```

Comment:
This field is already being set above in line 478. So you would have to remove that (which would be a breaking change).

Reply:
Ah that makes sense. Then I would move the `i.link(post_url.clone());` into an else branch to make this explicit.

Reply:
Like that?